backend/routes/group.py

```
@@ -0,0 +1,53 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from collections import defaultdict
+from models.user import User
+from models.post import Post
+from schemas.university import UniversityPage, Member, UniversityPost
+from database.session import SessionLocal
+
+router = APIRouter()
+
+def get_db():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_db`

(ANN201)